docs/developer-docs/integrations/sns/launching/launch-steps.md

```diff
@@ -1,47 +1,79 @@
-# SNS launch
+# SNS launch steps
+
 ## Overview
 An SNS can be launched in production by following the steps explained on a 
-high level [here](../lifecycle-sns/sns-launch.md).
+high level [here](../launching/launch-summary.md).
+
 Technically, these are the same steps that the
-[SNS local testing repository](../get-sns/local-testing.md) guides you through,
+[SNS local testing repository](../testing/local-testing.md) guides you through,
 with the difference that the commands target the canisters on the mainnet.
 
 To make the most important commands and what they need to look like for 
 mainnet more accessible, they are listed below.
 
-## Submitting an NNS proposal to approve the SNS {#SNS-launch-command-NNSproposal1}
-After preparations and choosing the parameters
-([Step 1: Preparation](../lifecycle-sns/sns-launch.md/#SNS-launch-step-preparation), 
-an [NNS proposal approves the creation of the SNS](#SNS-launch-step-NNSapproval).
-Anyone who owns and NNS neuron with enough stake can submit such a proposal
-that lists a principal wallet in SNS-W who can then deploy the SNS canisters.
-For the larger context, you can consider how this command is used in the SNS
-local testing repository
-[here](https://github.com/dfinity/sns-testing/blob/main/deploy_sns.sh#L18-L23).
-``` 
+## Requirememts
+
+1. IC SDK
+
+2. `ic-admin`
+
+3. `sns` CLI
+
+:::note
+The version of the sns CLI that is bundled with your dfx version may not have the latest commands described in the Usage section. If needed, it is recommended to build and use the sns CLI tool yourself.
+:::
+
+```bash
+git clone git@github.com:dfinity/ic.git
+cd ic
+bazel build //rs/sns/cli:sns
+ls bazel-bin/rs/sns/cli/sns 
+```
+
+## Steps
+
+- #### Step 1: Dapp developers choose the initial parameters of the SNS for a dapp
+
+- #### Step 2: Dapp developers submit NNS proposal so they can deploy to the SNS subnet
```

Comment:
Note that I think in the past we deliberately did not specify who needs to do this as this is really a NNS DAO decision, so everyone who is eligible on the NNS can send this proposal.
I don't have a strong preference here but wonder if it would be good to make sure this is aligned with language from Pete + co.